ArduCopter/RC_Channel.cpp

```diff
@@ -291,9 +292,12 @@ bool RC_Channel_Copter::do_aux_function(const AUX_FUNC ch_option, const AuxSwitc
 #endif
 
 #if AUTOTUNE_ENABLED == ENABLED
-        case AUX_FUNC::AUTOTUNE:
+        case AUX_FUNC::AUTOTUNE_MODE:
             do_aux_function_change_mode(Mode::Number::AUTOTUNE, ch_flag);
             break;
+        case AUX_FUNC::AUTOTUNE_SWITCH:
```

Comment:
How about renaming to AUTOTUNE_SAVE_SWITCH ?

Reply:
It don't save, it just allows testing of new or old gains. Save is still triggered on disarm. We could call it something more descriptive of course, the key thing was just to differentiate from the autotune mode function.

Reply:
changed to "AUTOTUNE_TEST_GAINS"